### ISSUE TITLE
Table sidebar items turn black on selection

### ISSUE THREAD

**Issue by robmaceachern** [jakob/Postico#604]
Selecting tables in the sidebar navigation causes the row to turn black. It seems to only stay black when the Structure or DDL control is selected. I am running Mojave dark mode.

![screen shot 2018-11-01 at 1 44 51 pm](https://user-images.githubusercontent.com/430436/47878789-58f82080-dddc-11e8-93d1-2ed66bd0d003.png)

Changing focus to another application seems to revert the rows back to normal.

## What software versions are you using?

Postico version:  
Version 1.4.2 (2845)

macOS version:  
10.14.1 (18B75)

PostgreSQL version:  
PostgreSQL 10.5 on x86_64-apple-darwin14.5.0, compiled by Apple LLVM version 7.0.0 (clang-700.1.76), 64-bit

**Comment by jakob** [comment]
Did you change the NSRequiresAquaSystemAppearance setting like this user:
https://github.com/jakob/Postico/issues/164#issuecomment-430774612

?

**Comment by robmaceachern** [comment]
No, I haven't ever (manually) altered those settings before. I ran `defaults delete -g NSRequiresAquaSystemAppearance` just to be sure and it seems to confirm that it wasn't set:
```
$ defaults delete -g NSRequiresAquaSystemAppearance
2018-11-03 10:27:59.513 defaults[2963:103957] 
Domain (kCFPreferencesAnyApplication) not found.
Defaults have not been changed.
```

**Comment by jakob** [comment]
Thanks for testing this. I've not seen this issue before, but it may be related to the fact that Postico 1.4.2 was built against the 10.13 SDK. Mojave introduced a lot of display bugs. The next bugfix update is going to be built against 10.14 SDK

Could you try the following beta version, and see if you still have this issue?
https://eggerapps-downloads.s3.amazonaws.com/postico-3735.zip
